apps/omg/lib/omg/state/transaction/signed.ex

```diff
@@ -77,6 +80,8 @@ defmodule OMG.State.Transaction.Signed do
             |> Enum.into(%{}, fn {witness, idx} -> {idx, witness} end)}
   end
 
+  defp get_witnesses_from_raw_tx(%Transaction.FeeTokenClaim{}, _raw_witnesses), do: {:ok, %{}}
```

Comment:
hm, is this branching on the tx type necessary here? if the FeeTokenClaim doesn't provide any witnesses, it should result in `{:ok, %{}}`, doesn't it? This module should at all cost remain raw_tx_type agnostic

Reply:
I witnessed issues pushing fee tx through the old pipeline. Will try to resolve differently.